src/linux/cpu.rs

```diff
@@ -531,6 +531,202 @@ pub(crate) fn get_physical_core_count() -> Option<usize> {
     Some(core_ids_and_physical_ids.len())
 }
 
+fn get_arm_implementer(implementer: u16) -> Option<&'static str> {
+    Some(match implementer {
+        0x41 => "ARM",
+        0x42 => "Broadcom",
+        0x43 => "Cavium",
+        0x44 => "DEC",
+        0x46 => "FUJITSU",
+        0x48 => "HiSilicon",
+        0x49 => "Infineon",
+        0x4d => "Motorola/Freescale",
+        0x4e => "NVIDIA",
+        0x50 => "APM",
+        0x51 => "Qualcomm",
+        0x53 => "Samsung",
+        0x56 => "Marvell",
+        0x61 => "Apple",
+        0x66 => "Faraday",
+        0x69 => "Intel",
+        0x70 => "Phytium",
+        0xc0 => "Ampere",
+        _ => return None,
+    })
+}
+
+fn get_arm_part(implementer: u16, part: u16) -> Option<&'static str> {
```

Comment:
Can you add a comment in the code saying where you got the value (the link you posted in the PR comment 😉 ) please?